test/mocks/MockV4Router.sol

```diff
@@ -38,4 +38,6 @@ contract MockV4Router is V4Router, ReentrancyLock {
     function _msgSender() internal view override returns (address) {
         return _getLocker();
     }
+
+    receive() external payable {}
```

Comment:
whats the motivation for this?

Reply:
just so that the mock contract can receive ETH to test SETTLE_WITH_BALANCE with ETH as the currency. The mock didnt previously have a way to receive ETH to do so